include/wx/string.h

```diff
@@ -395,11 +398,23 @@ class WXDLLIMPEXP_BASE wxString
 
 #else // wxUSE_UNICODE_UTF8
 
+#ifndef wxNO_UNSAFE_WXSTRING_CONV2
   static wxScopedCharBuffer ImplStr(const char* str,
                                     const wxMBConv& conv = wxConvLibc)
+#else
+  // conv must be explicit
```

Comment:
Again, I'd rather define some `wxSTRING_DEFAULT_CONV_ARG` macro, which would expand to either nothing or `= wxConvLibs`, than duplicating the function declarations.